### ISSUE TITLE
react/jsx-indent not detecting incorrect indentation

### ISSUE THREAD

**Issue by blissdev** [yannickcr/eslint-plugin-react#1111]
Given a JSX expression like ([gist](https://gist.github.com/blissdev/a4c601420b2bd4dcbbc954a3745903b8#file-jsx-indent-example-js)) where line 12 is obviously indented incorrectly:

```js
class ExampleComponent extends React.Component {
    render() {
        let app = this.props.data;

        return (
          <a href={app.url} className="item">
            <div className="form-container">
              <div className="top">
                <header>
                  <div className="title">
                    {app.thing.name}
             </div>
                </header>
              </div>
              <div className="attributes">
                <div className="attribute">
                  <div className="label">status</div>
                  <div className="value">{app.status}</div>
                </div>
              </div>
            </div>
          </a>
        );
    }
}

```

and with the [given config](https://gist.github.com/blissdev/a4c601420b2bd4dcbbc954a3745903b8#file-eslintrc-json) which has 2 space jsx indentation specified, there are no errors reported with the indentation. I can't figure out what the issue is. Is this a bug?

**Comment by dnalborczyk** [comment]
I can't reproduce this, I get an error.

I'm using eslint **3.19.0**, eslint-plugin-react **6.10.3**.

**Comment by ljharb** [comment]
Closing pending a repro.
